haskell-tests/Generators.hs

```diff
@@ -180,9 +180,9 @@ genPayloadUnsafe =
           genPayloadUpdateBakerStake,
           genPayloadUpdateBakerRestateEarnings,
           genPayloadUpdateBakerKeys,
-          genPayloadConfigureBaker,
           genPayloadConfigureDelegation
         ]
+            ++ [genPayloadConfigureBaker pv | pv <- [P4, P5, P6, P7, P8]]
```

Comment:
Is there a reason there is no `Enum` instance for `ProtocolVersion`?

Reply:
Probably not a particularly good one. It might have to do with the singletons stuff, at least historically.